src/Catalyst.Core.Modules.Web3/ApiModule.cs

```diff
@@ -183,5 +196,30 @@ public void Configure(IApplicationBuilder app)
                 app.UseSwaggerUI(swagger => { swagger.SwaggerEndpoint("/swagger/v1/swagger.json", "Catalyst API"); });
             }
         }
+
+        private sealed class SharedContainerProviderFactory : IServiceProviderFactory<ContainerBuilder>
+        {
+            readonly IContainer _container;
+
+            public SharedContainerProviderFactory(IContainer container)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private constructor 'SharedContainerProviderFactory'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979613)